web/cypress/integration/route/online-debug.spec.js

```diff
@@ -54,6 +54,9 @@ context('Online debug', () => {
       cy.get(domSelector.uriInput).type(`${defaultSettings.serveUrlMap[SERVE_ENV].split('//').pop()}${uri}`);
       cy.contains(routeLocaleUS['page.route.button.send']).click();
       // should not show the notification
+      
+      // test ci failed 
+      cy.wait(500)
```

Comment:
Could we use `interceptor` to check if that request is finished?

Reply:
> Could we use `interceptor` to check if that request is finished?

In fact, this test case does not have to wait for the response of the request, because It only needs to validate the url
and it passed in my local env. 

However, it is possible that multiple requests are made frequently and errors are reported because of blocking. PLS let me try to find out the exact cause of the problem first, then I'll try `interceptor` ^_^

Reply:
The regular expression does not take into account the case of `localhost`, that is why CI failed🤣, I will update soon, thanks all your help @LiteSun @juzhiyuan